packages/components/src/utils/validation/throwIfComponentIsDeprecated.ts

```diff
@@ -0,0 +1,5 @@
+import { getTagName } from '../../utils';
+
+export const throwIfComponentIsDeprecated = (host: HTMLElement, message: string): void => {
```

Comment:
That's not throwing anything, also what's the benefit of wrapping `console.warn` in another function that basically combines 2 parameters with a `:`?

Reply:
used this function because it was used in link social already for deprecated warning